distributed/worker.py

```diff
@@ -2680,6 +2681,13 @@ def get_call_stack(self, comm=None, keys=None):
         result = {k: profile.call_stack(frame) for k, frame in frames.items()}
         return result
 
+    def _notify_transition(self, key, start, finish, **kwargs):
+        for plugin in self.plugins.values():
+            try:
+                plugin.transition(key, start, finish, **kwargs)
+            except Exception:
+                logger.info("Plugin failed with exception", exc_info=True)
```

Comment:
We might want to include the name of the plugin in the logs.  Perhaps you could iterate over `name, plugin in self.plugins.items()` above and use the name here?